vin/app/controllers/vin/notes_controller.rb

```diff
@@ -1,77 +0,0 @@
-class Vin::NotesController < ApplicationController
-respond_to :json
-
-  # GET /vin/sub/:uid/shipments/:sid/notes
-  def index
-
-    @subscriber = Subscriber.find(params[:sub_id])
-
-    @package = @subscriber.shipments.find(params[:shipment_id]).package
-
-    notes = @package.note
```

Comment:
At the beggining i was thinking they might be multiple notes. I'm actually not sure yet, i sent an email asking the "client". That's the reason the response is wrapped in an array